app/server/utils.py

```diff
@@ -0,0 +1,10 @@
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+
+def log_ops_message(client, message):
+    channel_id = "C0388M21LKZ"
```

Comment:
For the future, we could probably put this in an environment variable so that it is not hard coded. But for future us.

Reply:
Absolutely, I admit I wasn't fully sure which channel this was for so I'll mark as future improvements!